lib/src/controllers/player_controller.dart

```diff
@@ -324,6 +324,13 @@ class PlayerController extends ChangeNotifier {
     PlatformStreams.instance.playerControllerFactory.clear();
   }
 
+  /// This method is to pause all players at once.
+  ///
+  /// This method will only pause the players it will not release the resources.
+  Future<void> pauseAllPlayers() async {
+    await AudioWaveformsInterface.instance.pauseAllPlayers();
```

Comment:
Please return the boolean here so that users can check if function executed successfully also update `stopAllPlayers` with same and state in the docs that what boolean represents.